### PR TITLE
Add SIEM and SOAR data retention periods

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds new sections on data retention:
- https://help.sumologic.com/docs/cse/automation-service/about-automation-service/#data-retention
- https://help.sumologic.com/docs/cloud-soar/overview/#data-retention

The new content was created by @vosorioparis. @jpipkin1 checked it in for him.

Issue number: [CSOAR-233](https://jira.kumoroku.com/jira/browse/CSOAR-233)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cloud-soar/overview.md [hunk 1]
```diff
@@ -36,3 +36,39 @@ Automated workflow processes help analysts function at an optimal level and redu
 
 ### Comprehensive security portfolio
 Cloud SOAR is part of Sumo Logic’s comprehensive security portfolio including security monitoring and analytics and Cloud SIEM, to help your organization modernize security operations.
+
+## Support and complaince
```
Comment (vosorioparis) [inline docs/cloud-soar/overview.md:40]: compliance

File: docs/cloud-soar/overview.md [hunk 2]
```diff
@@ -36,3 +36,39 @@ Automated workflow processes help analysts function at an optimal level and redu
...
+
+For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. Please keep in mind that if a customer needs to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
+
+| Data type | Retention period |
+| :-- | :-- |
+| Incidents | 7 years |
+| Triage | 7 years |
+| Entities | 7 years |
+| Playbook and action executions | 7 years |
+| CSE Records | Records (normalized logs) are stored in the partitions whose names begin with the string `sec_records`. There is one partition for each Record type. <br/><br/>There is no additional charge for storage of Records.| 90 days |
```
Comment (vosorioparis) [inline docs/cloud-soar/overview.md:65]: Sorry my bad, three last rows belong to CSE, can you please remove them?

File: docs/cloud-soar/overview.md [hunk 3]
```diff
@@ -36,3 +36,39 @@ Automated workflow processes help analysts function at an optimal level and redu
 
 ### Comprehensive security portfolio
 Cloud SOAR is part of Sumo Logic’s comprehensive security portfolio including security monitoring and analytics and Cloud SIEM, to help your organization modernize security operations.
```
[the remark this answers is not in the document]
Comment (vosorioparis) [reply docs/cloud-soar/overview.md:38]: Suggestion:

Cloud SOAR comprises both the Automation Service, which allows our Cloud SIEM to leverage the power of automated playbooks, and the full Cloud SOAR which includes case management among many other capabilities aimed at helping your organization modernize security operations

[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cloud-soar/overview.md]: @vosorioparis, I used your suggested sentence, and broke it into two:
> Cloud SOAR comprises both the Automation Service, which allows our Cloud SIEM to leverage the power of automated playbooks, and the full Cloud SOAR. Cloud SOAR combines automation with case management, among many other capabilities aimed at helping your organization modernize security operations.

File: docs/cloud-soar/overview.md (hunk 2 above)
Comment (jpipkin1) [reply docs/cloud-soar/overview.md]: I removed the rows "CSE Records", "CSE Signals", and "CSE Insights".

File: docs/cloud-soar/overview.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cloud-soar/overview.md]: Good catch! Fixed.

**Comment by jpipkin1** [comment]
Thanks for the review, @vosorioparis. I made your suggested changes.

File: docs/cloud-soar/overview.md [hunk 4]
```diff
@@ -35,4 +35,37 @@ Cloud SOAR acts as the connective tissue between your existing tools to automate
...
+We automatically delete the following customer data according to the table retention period below, except for clients that need to ensure HIPAA compliance.
+
+| Data type | Retention period |
+| :-- | :-- |
+| Incidents | 2 years |
+| Triage | 2 years |
+| Entities | 2 years |
+| Playbook and action executions | 2 years |
+
+For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. Please keep in mind that if a customer needs to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```
Comment (kimsauce) [inline docs/cloud-soar/overview.md:57]:
```suggestion
For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```

File: docs/cloud-soar/overview.md [hunk 5]
```diff
@@ -35,4 +35,37 @@ Cloud SOAR acts as the connective tissue between your existing tools to automate
...
+| :-- | :-- |
+| Incidents | 7 years |
+| Triage | 7 years |
+| Entities | 7 years |
+| Playbook and action executions | 7 years |
+
+#### Custom retention periods
+
+Customers can request retention period times different from those declared in the tables above, as long as the retention
+period requested is greater than 1 day yet less than 5000 days.
```
Comment (kimsauce) [inline docs/cloud-soar/overview.md:69]:
```suggestion
You can request retention period times different from those declared in the tables above, as long as the retention period requested is greater than 1 day yet less than 5000 days.
```

File: docs/cloud-soar/overview.md [hunk 6]
```diff
@@ -35,4 +35,36 @@ Cloud SOAR acts as the connective tissue between your existing tools to automate
...
+We automatically delete the following customer data according to the table retention period below, except for those that need to ensure HIPAA compliance.
+
+| Data type | Retention period |
+| :-- | :-- |
+| Incidents | 2 years |
+| Triage | 2 years |
+| Entities | 2 years |
+| Playbook and action executions | 2 years |
+
+For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```
Comment (jpipkin1) [inline docs/cloud-soar/overview.md:57]:
```suggestion
For those that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```

File: docs/cse/automation-service/about-automation-service.md [hunk 7]
```diff
@@ -177,3 +178,36 @@ The Sumo Logic Terraform provider also supports automations. For more informatio
 The Automation Service uses the Cloud SOAR API. For more information about the API, click the **?** button in the upper right of the Cloud SOAR UI to see the API Documentation manual.
 :::
 
+### Data retention
+
+This section lists the retention period for each type of data generated. 
+
+#### Default retention periods by data type
+
+We automatically delete the following customer data according to the table retention period below, except for clients that need to ensure HIPAA compliance.
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cse/automation-service/about-automation-service.md]: Fixed.

File: docs/cse/automation-service/about-automation-service.md [hunk 8]
```diff
@@ -177,3 +178,36 @@ The Sumo Logic Terraform provider also supports automations. For more informatio
...
+| CSE Signals | Stored in the `sec_signal` partition.<br/><br/>There is no additional charge for storage of Signals. | 2 years |   
+| CSE Insights | The `sumologic_system_events` partition contains Insights and Insight-related events that result from system actions. <br/><br/>The `sumologic_audit_events` partition contains Insights and Insight-related events that result from user actions.<br/><br/>There is a charge for storage of Insight-related data in the audit indexes. Note however the volume of data is typically very low compared to log ingestion levels. | By default, these partitions have a retention period of 30 days. This period is [customer-configurable](/docs/get-started/help#support). |
+
+#### Custom retention periods
+
+Customers can request retention period times different from those declared in the tables above, as long as the retention
+period requested is greater than 1 day yet less than 5000 days.
+
+In order, to do that please open a [Support ticket](/docs/get-started/help#support) with your request.
```
Comment (kimsauce) [inline docs/cse/automation-service/about-automation-service.md:213]:
```suggestion
In order to do that, please open a [Support ticket](/docs/get-started/help#support) with your request.
```

File: docs/cse/automation-service/about-automation-service.md [hunk 9]
```diff
@@ -177,3 +178,36 @@ The Sumo Logic Terraform provider also supports automations. For more informatio
 The Automation Service uses the Cloud SOAR API. For more information about the API, click the **?** button in the upper right of the Cloud SOAR UI to see the API Documentation manual.
 :::
 
+### Data retention
+
+This section lists the retention period for each type of data generated. 
+
+#### Default retention periods by data type
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cse/automation-service/about-automation-service.md:185]: @kimsauce, I think these tables are okay. The first table is for those who don't need to ensure HIPAA compliance (2 years retention), and the second table is for those do do need to ensure HIPAA compliance (7 years retention).

File: docs/cloud-soar/overview.md [hunk 10]
```diff
@@ -35,4 +35,36 @@ Cloud SOAR acts as the connective tissue between your existing tools to automate
...
+We automatically delete the following customer data according to the table retention period below, except for those that need to ensure HIPAA compliance.
+
+| Data type | Retention period |
+| :-- | :-- |
+| Incidents | 2 years |
+| Triage | 2 years |
+| Entities | 2 years |
+| Playbook and action executions | 2 years |
+
+For those that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```
Comment (kimsauce) [inline docs/cloud-soar/overview.md:57]:
```suggestion
For HIPAA-compliant customers, we delete data following the retention periods below. 

:::info
If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
:::
```

**Comment by jpipkin1** [comment]
Thanks, @kimsauce! Your suggestions look great.

**Pull request merged**